bears/r/FormatRBear.py

```diff
@@ -20,12 +20,12 @@ class FormatRBear:
 
     @staticmethod
     def create_arguments(filename, file, config_file,
-                         r_keep_comments: bool=True,
+                         r_keep_comments: bool=None,
```

Comment:
humm... i didn't think about that?

2016-06-18 19:47 GMT+01:00 Mischa Krüger <notifications@github.com>:

> In bears/r/FormatRBear.py
> <https://github.com/coala-analyzer/coala-bears/pull/525#discussion_r67604912>
> :
>
> > @@ -20,12 +20,12 @@ class FormatRBear:
> >
> >      @staticmethod
> >      def create_arguments(filename, file, config_file,
> > -                         r_keep_comments: bool=True,
> > +                         r_keep_comments: bool=None,
>
> you think we should make everything None and use the default style of
> formatR? I fear that the default style of formatR might change depending
> on version^^
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/coala-analyzer/coala-bears/pull/525/files/d84aca049bf7210b89f225a17fcc69289c4832c8#r67604912>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AJ9xoj59xMvPCZcfOASRjkH_TBmOawOKks5qND0ngaJpZM4I14vI>
> .
>